build.gradle

```diff
@@ -1038,42 +1038,40 @@ def buildTime() {
   return df.format(new Date())
 }
 
-// Takes the version, and if -SNAPSHOT is part of it replaces SNAPSHOT
-// with the git commit version.
 @Memoized
 def calculateVersion() {
-  String version = rootProject.version
-  if (version.endsWith("-SNAPSHOT")) {
-    version = version.replace("-SNAPSHOT", "-dev-" + getCheckedOutGitCommitHash())
-  }
-  return version
+  def gitDetails = getGitCommitDetails(10) // Adjust length as needed
+  return "${gitDetails.date}-develop-${gitDetails.hash}"
```

Comment:
@jflo thinking about the impact of this on docker builds.  we would probably have a build explosion if we published versions like this on dockerhub.  Does the new CI pipeline explicitly push versions into snapshot builds on `main` ?  or is that something we need to account for here?

Reply:
I disabled the duplicate circle-ci workflows in this PR that are causing us to publish docker images to dockerhub using snapshot build tags

Reply:
reverted the change in favor of just disabling the circle-ci webhook

Reply:
you can still have build artifacts, just not docker images:  https://github.com/hyperledger/besu/releases/tag/develop